### PR TITLE
Update _results-key.md

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: toolkit/assessments/interview-results-key.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ section: hiring-phases
 sidenav: hiring-phases
 phase: assessments
 title: Interview Results Key
```
Comment (stephaniefaith) [inline toolkit/assessments/interview-results-key.md:7]:
```suggestion
title: Assessments Results Key
```

**Review by stephaniefaith — CHANGES_REQUESTED** [review]
one thing you missed before you merge.

**Comment by stephaniefaith** [comment]
@wslack ^

**Comment by wslack** [comment]
fixed per Stephanie

**Pull request merged**